src/mono/mono/mini/mini.c

```diff
@@ -961,61 +960,7 @@ mini_assembly_can_skip_verification (MonoDomain *domain, MonoMethod *method)
 static gboolean
 mini_method_verify (MonoCompile *cfg, MonoMethod *method, gboolean fail_compile)
 {
-	GSList *tmp, *res;
```

Comment:
The whole function can be removed.

Reply:
Ok, removed `mini_method_verify` and `mono_compile_is_broken` and fixed up the callers